tests/base/model_valid_epoch_ends.py

```diff
@@ -52,3 +52,39 @@ def _mean(res, key):
 
         results = {'progress_bar': pbar, 'log': logs}
         return results
+
+    def validation_epoch_end__multiple_dataloaders(self, outputs):
```

Comment:
I added this method, but now since you merged master there is actually already a version like this  (see above), let's just use that one instead and deleted this one.
maybe it solves the failing test

Reply:
yeah!